P1/Project_Code/frontend/src/components/Media_Cards/MediaCard.jsx

```diff
@@ -11,38 +10,34 @@ const MediaCard = ({ movie, type }) => {
         setImgSrc(imagePath);
     }, [movie]);
 
+    if (!imgSrc) {
```

Comment:
recommend following the if-else pattern to render the other 2 layouts instead of a ternary statement for easier redability.